gender_novels/corpus_gen.py

```diff
@@ -7,25 +7,30 @@
 import pywikibot
```

Comment:
Please remove pywikibot from the general imports until there are installation instructions for the module. (There aren't right now, correct?). You can then import the package directly in the two functions that actually use it. That way, people who don't have pywikibot installed can still use this file--just not the two wiki functions.

Reply:
It should work now with the user-config.py I pushed to the repo, and I'd like to know if it doesn't work on other people's machines.

Reply:
Ah. I hadn't seen that new file. 
Down the road, can you see if there is a way to just hardcode the 3 lines of config into corpus_gen.py? Having just three lines of config for an external module in a python file in our main directory is--irksome. But I understand that it needs to be there for the moment.